File: 2st/Yeonwoo/문자열 섞기.py
def solution(str1, str2):
    L1 = []
    L2 = []
    L3 = []
    L1 = list(str1)
    L2 = list(str2)
    for i in range(0,len(str1)):
        L3.append(L1[i])
        L3.append(L2[i])
    answer = ''.join(L3)
    return answer

File: 4st/Gau/programmer/문자열 섞기.java
class Solution {
    public String solution(String str1, String str2) {
        String a = "";
        
        for(int i = 0; i < str1.length(); i++) {
        	a += str1.charAt(i);
        	a += str2.charAt(i);
        }
        return a;
    }
}